app/src/components/user/AccountIndexPage/ManageAccount/ManageTeams/TeamViewer/index.js

```diff
@@ -103,6 +104,7 @@ const TeamViewer = ({ teamId }) => {
           defaultActiveKey="0"
           items={manageWorkspaceItems}
           className="manage-workspace-tabs"
+          onChange={trackWorkspaceSettingToggled}
```

Comment:
use the `activeKey` present in `onChange` to also send label of the tab that the user switched to